tensorflow/python/ops/sparse_ops.py

```diff
@@ -2732,6 +2733,99 @@ def sparse_transpose(sp_input, perm=None, name=None):
     return transposed_st
 
 
+@tf_export("sparse.map_values")
+@dispatch.add_dispatch_support
+def map_values(op, *args, **kwargs):
```

Comment:
Maybe call this `map_explicit_values` or `map_nondefault_values`, to make it more clear that this only applies to explicit values?  (For users who are less knowledgable about the inner workings of SparseTensor, it may not be clear that a SparseTensor's "values" just refers to the explicit values.)

@alextp Thoughts on naming for this?